model/src/main/java/com/redhat/ipaas/model/connection/Connection.java

```diff
@@ -49,7 +50,7 @@ default String getKind() {
 
     Optional<String> getConnectorId();
 
-    String getConfiguredProperties();
+    Map<String, String> getConfiguredProperties();
```

Comment:
At the end of the the day, it comes form an textual input box, and camel encodes it into a URL string.  I could open it up, but sounds like thats adding some complexity without much value.